init-odkx-sync-endpoint.py

```diff
@@ -32,11 +39,48 @@ def run_interactive_config():
     typer.echo("We'll need some information from you to get started though...")
     time.sleep(1)
     typer.echo("")
+
+    allow_default = typer.confirm("Do you wish to use default configuratilon?", default=True)
+    typer.echo("")
+
+    if not allow_default:
+        if is_cache_present():
+            allow_cache = typer.confirm("Do you wish to use cached configuratilon?", default=True)
+            typer.echo("")
+            if allow_cache:
+                cached_data = load_progress()
+                enforce_https = json.dumps(cached_data['enforce_https'])
+                manual_certificate = json.dumps(cached_data['manual_certificate'])
+                if manual_certificate : 
+                    setup_certificate(env) 
+                else:
+                    skip_setup_certificate(env)
+
+            else:
+                enforce_https = run_cache_setup(env)
+        else:
+            enforce_https = run_cache_setup(env)
+    else :
+        typer.echo("Please enter your system Password")
+        enforce_https = json.dumps(default_config['enforce_https'])
+        skip_setup_certificate(env)
```

Comment:
It looks like the default sets enforce https to be true, but then runs the certbot setup with the default parameters of whatever is in the env file, which will not be valid unless the user actually inputs their email and domain at some point.

Reply:
hey @matt9j 

I don't think i get your point pretty well, however i think you are scared of scenarios when `env` is blank  but yet https is enabled.  I don't think we shall experience such scenarios in our setup process because the script will never execute unless the `env` is initialised correctly.  This is the main reason why I left the initial functionality of setting up the `env` untouched like [here](https://github.com/odk-x/sync-endpoint-default-setup/blob/outreachy-2022/init-odkx-sync-endpoint.py#L23)

Reply:
unless the env variable doesn't satisfy all use cases.